### ISSUE TITLE
All unicode comes into tranliteration

### ISSUE THREAD

**Issue by DaZui** [git-for-windows/git#725]
- [x] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen)
   or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue
   matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? 32-bit or 64-bit? Include the
   output of `git version` as well.

2.8.1 x64

```
$ git --version
git version 2.8.1.windows.1
```

 - Which version of Windows are you running? 32-bit or 64-bit?

Windows 10 10586 64 bit

 - What options did you set as part of the installation? Or did you choose the
   defaults?

defaults

### Details

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

git-bash works well, CMD went wrong.

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
ls
```
 - What did you expect to occur after running these commands?

Correct expression of unicode character

こんにちは  你好

 - What actually happened instead?

''$'\343\201\223\343\202\223\343\201\253\343\201\241\343\201\257'  ''$'\344\275\240\345\245\275'

![image](https://cloud.githubusercontent.com/assets/5781966/14410947/f0400eb6-ff00-11e5-995c-d0582da57c63.png)

**Comment by DaZui** [comment]
OK. It seems that only happened in Git 2.8.0 and 2.8.1. 2.7.4 Works well. Maybe something wrong?

**Comment by dscho** [comment]
The `ls` utility is part of MSYS2's `coreutils` package. @DaZui why don't you install [`MSYS2`](https://msys2.github.io/), verify that it displays the same behavior, and then report the issue [in upstream MSYS2](https://github.com/Alexpux/MSYS2-packages/issues)?

**Comment by dscho** [comment]
FWIW 2.8.0 came with coreutils 8.25-1, 2.7.4 came with coreutils 8.24-1.

**Comment by dscho** [comment]
I'll give this ticket a couple more days. If there is still no feedback, it will be closed.

**Comment by DaZui** [comment]
![image](https://cloud.githubusercontent.com/assets/5781966/14803865/4cfed938-0b23-11e6-88b0-9720067d944a.png)

MSYS2 is working well.

**Comment by dscho** [comment]
@DaZui please do not force me to *beg* for more information. Which version of coreutils did you test there? If you copy `ls.exe` from MSYS2 to Git for Windows, does it fix the issue? Etc etc.

**Comment by ishisaka** [comment]
If it's on bash.exe, it's indicated right, but if it's on cmd.exe, it's indicated as a report.


![ls_cmd](https://cloud.githubusercontent.com/assets/589304/15917511/5b0eb786-2e38-11e6-846f-4bae1ad2ca87.png)

![ls_bash](https://cloud.githubusercontent.com/assets/589304/15917512/6036cc58-2e38-11e6-9c84-fccb919fbad5.png)

**Comment by dscho** [comment]
@ishisaka that is valuable information! Thank you!

To allow me to reproduce swiftly (between my other duties), would you kindly prepare a tiny repository for me to clone, a repository that contains the indicated file name?

**Comment by ishisaka** [comment]
Please experiment on the following procedure.

It can be operated by Windows of ENU version.

1. Setup ConEmu(Other Console Emulator)
2. The font is changed to "MS Gothic". The following figure referring.

![snapcrab_noname_2016-6-9_19-58-41_no-00](https://cloud.githubusercontent.com/assets/589304/15927429/9bc8f540-2e7c-11e6-912e-04b757646c08.png)

3． The clone does the following URL repository.

<https://github.com/ishisaka/gitforwindows_ls_test.git>

4． Open CMD.EXE on ConEmu

5. Change Directory To Clone Directory.

6． Run ls.exe

![snapcrab_noname_2016-6-9_20-4-29_no-00](https://cloud.githubusercontent.com/assets/589304/15927568/76431930-2e7d-11e6-9ba2-cf64ee2cc4e6.png)

NG.

6．Open bash.exe on ConEmu

7. Run ls.exe

![snapcrab_noname_2016-6-9_20-6-15_no-00](https://cloud.githubusercontent.com/assets/589304/15927622/b114c4e6-2e7d-11e6-85a3-52fab85238bc.png)

OK.

**Comment by dscho** [comment]
@ishisaka thank you for the repository.

I do not have ConEmu installed, and neither do I plan to do that. I had limited success reproducing your scenario in Git CMD, though.

This is what I found: if the environment setting `LANG=en_US.UTF-8` is not set, I get the same behavior as you described. If it is set, I get rectangles instead (I guess that I would need the Japanese language pack to see Japanese characters instead).

What is the value of the environment variable `LANG` in your working case?
